python/py-isort/files/isort27

```diff
@@ -0,0 +1,2 @@
+bin/isort-2.7
+bin/isort-identify-imports-2.7
```

Comment:
the versions for PY27/PY35 are pinned to the latest versions supported by upstream. However, these do not contain the console script `isort-identify-imports`. So the select files for these two versions should have a `-` on the second line. 

Additionally, since the revisions are set separately in the PY27/PY35 subports you will need to increase the `revision` there as well. 

Sorry that I missed these two things in the earlier review - other than that it should be good to go; thanks again!

Reply:
Done